ui/src/containers/Resources/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {
+  Gallery,
+  GalleryItem,
+  Card,
+  CardBody,
+  Pagination,
+  GridItem,
+  Grid
+} from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
+import './Resources.css';
+
+const Resources = () => {
+  const [pageNumber, setPageNumber] = React.useState(1);
+  const [perPgae, setPerPage] = React.useState(20);
+
+  const setPage = (event: React.MouseEvent | React.KeyboardEvent | MouseEvent, page: number) => {
+    setPageNumber(page);
+  };
+
+  const perPageSelect = (
+    event: React.MouseEvent | React.KeyboardEvent | MouseEvent,
+    perpage: number
+  ) => {
+    setPerPage(perpage);
+  };
+  return (
+    <React.Fragment>
+      <Grid>
+        <GridItem>
+          <Pagination
+            itemCount={200}
+            perPage={perPgae}
```

Comment:
are we going to include pagination in this release ? (if not then probably we can remove this for now)

Reply:
Yes in the follow up pr it is been removed